ush/set_rrfs_config_SDL_VDL_MixEn.sh

```diff
@@ -30,6 +35,6 @@ elif [[ ${ngvarloc} == "2" ]] && [[ ${nsclgrp} == "2" ]]; then
   else
     ens_h="328.632,82.1580,4.10790,4.10790,82.1580"
     ens_v="-0.30125,-0.30125,-0.30125,-0.30125,0.0"
-    i_ensloccov4var=1
+    r_ensloccov4var=0.05
```

Comment:
@shoyokota This is also a general question. The SDL/VDL will turn off readin_localization. However, do you expect that SDL/VDL will further benefit from a height-dependent localization radius? Thanks!

Reply:
We can expect the benefit from a height-dependent localization length also for SDL/VDL, but I guess the benefit is smaller than the case without SDL/VDL and optimization becomes complicated more. So far, I prefer not to use a height-dependent localization in the case with SDL/VDL.

Reply:
@shoyokota Thanks for the discussion. You might have heard about another application other than RRFS, called 3D-RTMA (Real-time Mesoscale Analysis). 3DRTMA is also based on the ufs-srweather-app. It tries to get an atmospheric analysis closest to observations (especially surface observations). It has to use a smaller localization radius near the surface but some kind of larger radii away from the surface. We may also want to try SDL/VDL in 3DRTMA but would prefer a height-dependent SDL/VDL. We would like to seek your input/help at that time. Thanks!